package.json

```diff
@@ -1,69 +1,20 @@
 {
-  "name": "tudo",
-  "version": "1.0.2",
-  "description": "A single version of the truth with integrated task tracking, delegation and accountability.",
-  "main": "index.js",
+  "repository": {},
```

Comment:
we should copy _most_ of the `metadata` from the "old" `package.json` to here (_not the `scripts` or `dependencies`_)